tool/jt.rb

```diff
@@ -43,6 +43,22 @@
 METRICS_REPS = Integer(ENV['TRUFFLERUBY_METRICS_REPS'] || 10)
 DEFAULT_PROFILE_OPTIONS = %w[--cpusampler --cpusampler.Output=flamegraph]
 
+MRI_TEST_RETAG_RETAIN_PATTERNS = [
+  /hangs/i,
+  /slow/i,
+  /java\.lang\./,
```

Comment:
Is this for AssertionError and other exceptions?
I think we should be more explicit like `/java\.lang\.\w+(Error|Exception)/`
because `git grep -F java.lang. test/mri/excludes/` seems to match too much, e.g. `"Exception raised: Exception raised: <#<SystemStackError: stack level too deep from java.lang.StringCoding.implEncodeAsciiArray(StringCoding.java:77)>>"`

Also I'm thinking maybe we shouldn't retain those at all, similar thinking to ShouldNotReachHere.